lib/components/user-input/Textarea.tsx

```diff
@@ -34,6 +38,9 @@ export const Textarea = ({
   disclaimer,
   onBlur,
   onEditCompleted = noop,
+  defaultStyle = true,
+  className,
+  outerClassName = '',
```

Comment:
Not sure about this `outerClassName`. Why does the caller cannot set this around the component?